core/src/main/java/org/apache/accumulo/core/metrics/MetricsUtil.java

```diff
@@ -105,14 +108,16 @@ public DistributionStatisticConfig configure(Meter.Id id,
         }
       };
 
-      Class<? extends MeterRegistryFactory> clazz =
-          ClassLoaderUtil.loadClass(factoryClass, MeterRegistryFactory.class);
-      MeterRegistryFactory factory = clazz.getDeclaredConstructor().newInstance();
+      for (String factoryName : getTrimmedStrings(factoryClasses)) {
+        Class<? extends MeterRegistryFactory> clazz =
+            ClassLoaderUtil.loadClass(factoryName, MeterRegistryFactory.class);
```

Comment:
If someone had the old Factory interface configured it seems like this would blow up.

Reply:
Addressed in d8551b56e4